runners_manager/vm_creation/openstack.py

```diff
@@ -74,6 +75,12 @@ def create_vm(self, runner: Runner, runner_token: int or None,
             userdata=self.script_init_runner(runner, runner_token, github_organization,
                                              installer)
         )
+        while self.nova_client.servers.get(instance.id).status not in ['ACTIVE', 'ERROR']:
+            time.sleep(2)
+
+        if self.nova_client.servers.get(instance.id).status == 'ERROR':
+            logger.info('vm is got an error, try recreating')
```

Comment:
Let's adjust this error message:
```suggestion
            logger.info('VM creation failed, creating a new one')
```

Also does `self.nova_client.server.get(instance.id)` gives us a description of the error if any? Could be nice to print it in the logs as well.